app/controllers/user_controller.rb

```diff
@@ -0,0 +1,51 @@
+ServiceDiscovery::App.controllers :users do
+  
+  post '/:user_id/address' do
+    get_create_address_params
+    address = create_address @input, params[:user_id]
+    status 201
+    {address_id: address.id}.to_json
+  end
+
+  get '/:user_id/address' do
+    addresses = Address.where('user_id' => params[:user_id])
+    status 201
```

Comment:
Please correct the status. It should not be 201. 
I would suggest to not set it explicitly. It will go as 200 by default